java/matcher_test.go

```diff
@@ -199,3 +202,7 @@ func TestVulnerable(t *testing.T) {
 		})
 	}
 }
+
+func TestMatcher(t *testing.T) {
+	test.RunMatcherTests(zlog.Test(context.Background(), t), t, "testdata/matcher", new(Matcher))
```

Comment:
so when this runs, it keeps the current directory context? (as in, it finds `testdata/matcher` relative to the `java` directory and not the `test` directory. I didn't realize that, cool